go.mod

```diff
@@ -1,32 +1,191 @@
 module github.com/owncloud/ocis-hello
 
-go 1.16
+go 1.20
 
 require (
 	contrib.go.opencensus.io/exporter/jaeger v0.2.1
 	contrib.go.opencensus.io/exporter/ocagent v0.7.0
 	contrib.go.opencensus.io/exporter/zipkin v0.1.2
-	github.com/asim/go-micro/v3 v3.6.0
+	github.com/cs3org/go-cs3apis v0.0.0-20230516150832-730ac860c71d
+	github.com/cs3org/reva/v2 v2.13.4-0.20230526111208-d62d1497abb9
 	github.com/go-chi/chi v4.1.2+incompatible
-	github.com/go-chi/chi/v5 v5.0.4
-	github.com/go-chi/render v1.0.1
-	github.com/golang/protobuf v1.5.2
+	github.com/go-chi/chi/v5 v5.0.8
+	github.com/go-chi/render v1.0.2
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
 	github.com/micro/cli/v2 v2.1.2
 	github.com/oklog/run v1.1.0
 	github.com/openzipkin/zipkin-go v0.2.5
-	github.com/owncloud/ocis v1.11.0
-	github.com/prometheus/client_golang v1.11.0
+	github.com/owncloud/ocis/v2 v2.0.1-0.20230529065344-fd4301b0d71e
```

Comment:
This is a specific commit on an own branch inside ocis that does not have the `tests/acceptance/filesForUpload/'single'quotes.txt` file

The issue is that when I add ocis as a dependency 
`go get  github.com/owncloud/ocis/v2@v3.0.0-rc.4` gives me
`go: github.com/owncloud/ocis/v2@v3.0.0-rc.4: invalid version: module path includes a major version suffix, so major version must match`

And
`go get github.com/owncloud/ocis/v2@7094891f4de381102b05c6503751dc85d82c0782 #latest commit` gives me
```
go: downloading github.com/owncloud/ocis/v2 v2.0.1-0.20230530000306-7094891f4de3
go: github.com/owncloud/ocis/v2@7094891f4de381102b05c6503751dc85d82c0782: create zip: tests/acceptance/filesForUpload/'single'quotes.txt: malformed file path "tests/acceptance/filesForUpload/'single'quotes.txt": invalid char '\''
tests/acceptance/filesForUpload/new-'single'quotes.txt: malformed file path "tests/acceptance/filesForUpload/new-'single'quotes.txt": invalid char '\''
```

I found somewhere a bug-report for that, go can only work with certain file-names

The only way to do it I found is to make a commit on a new branch in ocis that does not have the problematic file. But what is the "right" thing to do?